docs/rules/prefer-power-assert.md

```diff
@@ -28,6 +28,6 @@ test(t => {
 import test from 'ava';
 
 test(t => {
-	t.ok(foo === bar);
+	t.truthy(foo === bar);
```

Comment:
@twada Do we want to continue using a truthy check or would it maybe be better to use `t.true` instead?